### ISSUE TITLE
Add create new deck type screen

### ISSUE THREAD

**Issue by doYouSpeakFish** [doYouSpeakFish/appOfManyDecks#11]
There should be a button on the decks list page that allows a new type of deck to be created. Tapping this should launch a screen that allows a user to create a new deck type. 

A new deck should have a title, description, and the option to add a cover image for the deck. There should be a button for adding a new card to the deck. Tapping this button should allow the user to enter a card name, description, and upload an image for the card.

The deck should be saved to the database.

depends on issue #6
